cms/static/js/base.js

```diff
@@ -67,6 +90,12 @@ domReady(function() {
     if ($.browser.msie) {
         $.ajaxSetup({ cache: false });
     }
+
+    //Initiate the edx tool kit dropdown menu
+    var dropdownMenuView = new DropdownMenuView({
+        el: '.js-header-user-menu'
+    });
+    dropdownMenuView.postRender();
```

Comment:
This runs even on pages not using the UI Toolkit dropdown menu. It would be good if it only ran when needed.